chia/_tests/wallet/db_wallet/test_db_graftroot.py

```diff
@@ -132,7 +132,14 @@ def filter_none(values: list[bytes32]) -> list[bytes32]:
                 )
                 new_final_bundle = WalletSpendBundle([new_fake_spend, graftroot_spend], G2Element())
                 result = await sim_client.push_tx(new_final_bundle)
-                assert result == (MempoolInclusionStatus.FAILED, Err.ASSERT_ANNOUNCE_CONSUMED_FAILED)
+
+                # The original version of this test made sure we return the
+                # following error. Now that puzzle reveal validation was moved
+                # into chia_rs, this failure will be detected before running
+                # the puzzle. This test probably needs to be updated
+                # assert result == (MempoolInclusionStatus.FAILED, Err.ASSERT_ANNOUNCE_CONSUMED_FAILED)
+
+                assert result == (MempoolInclusionStatus.FAILED, Err.WRONG_PUZZLE_HASH)
```

Comment:
same question as the other test return value

Reply:
yeah, I think this test is trying to test something else, and used to rely on the puzzle reveal being validated last